Mention the applied CSS styles in the documentation

wbadam opened the pull request:
Belongs to  #8395

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/framework/9219)
<!-- Reviewable:end -->

wbadam committed:
Mention the applied CSS styles in the documentation (#8395)

pleku commented:
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 76 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC1qtTbarAUOWtoEW0:-KjC1qtUvJ4NLbITeesK:b3zqyin) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L76)):*
> The drag source element, additional to it's primary style name, have a style name with the `-dragsource` suffix. For example, a Label component would have the style name `v-label-dragsource`.

When does it have the style name ???

Isn't it applied only when the drag has started ? Please clarify

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 76 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC2JkzerFaQKGrojW-:-KjC2JkzerFaQKGrojW0:bkfkw6e) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L76)):*
> `-dragsource`

Is it `-dragsource` and not `-drag-source` ?

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 127 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC2BWmpO6tZs5_UGKI:-KjC2BWmpO6tZs5_UGKJ:b-v4thye) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L127)):*
> the element accepts drops

(only if the drag over criteria passes)

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 222 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC2W6Im0WNTs4DwS3E:-KjC2W6Im0WNTs4DwS3F:b605hqf) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L222)):*
> Css

CSS

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 224 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC2_SYYqdpSG5hF8PO:-KjC2_SYYqdpSG5hF8PP:b1lrov5) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L224)):*
> A drag source Grid's rows have the `v-grid-row-draggable` style name applied to indicate that the rows are draggable.

Looking at the code it is actually `v-draggable` so and in the in progress PR #9091 it is "v-grid-row-dragsource` so what the hell ?

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 225 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC65RY3DrbQ9DW_wNO:-KjC65RY3DrbQ9DW_wNP:b7yn8ss) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L225)):*
> ```AsciiDoc
> 
> A drag source Grid's rows have the `v-grid-row-draggable` style name applied to indicate that the rows are draggable.
> ```

What happens to the rows that are _actually being dragged_, are those themable and distinctable from the non-dragged rows ?

There seems to be the `-drag-badge`, what should that be used for ?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/9219)*
<!-- Sent from Reviewable.io -->

wbadam committed:
Make some fixes according review

wbadam commented:
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 76 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC1qtTbarAUOWtoEW0:-KjCAUTGZqZyh5e-Oyfx:bz11rs5) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L76)):*
<details><summary><i>Previously, pleku (Pekka Hyvönen) wrote…</i></summary><blockquote>

> The drag source element, additional to it's primary style name, have a style name with the `-dragsource` suffix. For example, a Label component would have the style name `v-label-dragsource`.

When does it have the style name ???

Isn't it applied only when the drag has started ? Please clarify
</blockquote></details>

No it is applied when the extension is applied, removed when the extension is removed.

There is no style for 'component is being dragged', may make sense to add one.

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 76 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC2JkzerFaQKGrojW-:-KjCBJMtJx7EtabTvzgj:bwjdu07) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L76)):*
<details><summary><i>Previously, pleku (Pekka Hyvönen) wrote…</i></summary><blockquote>

> `-dragsource`

Is it `-dragsource` and not `-drag-source` ?
</blockquote></details>

It is and my argument was that `-` would mean a changing suffix such as `-top` and `-bottom` and here `-source` is not a suffix.
Could change if `-drag-source` would be better.

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 127 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC2BWmpO6tZs5_UGKI:-KjCC2RRmFQ2boDT_ZKW:b-896fix) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L127)):*
<details><summary><i>Previously, pleku (Pekka Hyvönen) wrote…</i></summary><blockquote>

> the element accepts drops

(only if the drag over criteria passes)
</blockquote></details>

Done.

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 222 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC2W6Im0WNTs4DwS3E:-KjCC74rnpmUMnfKBjeS:b-896fix) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L222)):*
<details><summary><i>Previously, pleku (Pekka Hyvönen) wrote…</i></summary><blockquote>

> Css

CSS
</blockquote></details>

Done.

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 224 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC2_SYYqdpSG5hF8PO:-KjCCPsIjrZZhegIvVPh:b-fm50gh) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L224)):*
<details><summary><i>Previously, pleku (Pekka Hyvönen) wrote…</i></summary><blockquote>

> A drag source Grid's rows have the `v-grid-row-draggable` style name applied to indicate that the rows are draggable.

Looking at the code it is actually `v-draggable` so and in the in progress PR #9091 it is "v-grid-row-dragsource` so what the hell ?
</blockquote></details>

Did I misunderstand that you wanted the documentation reflect PR #9091?

Before PR it is `v-draggable`, after PR it is `v-grid-row-dragsource`, fixed the docs to the latter.

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 225 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC65RY3DrbQ9DW_wNO:-KjCD5KZiwEI6Wo0-2K-:bb4zfqr) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L225)):*
<details><summary><i>Previously, pleku (Pekka Hyvönen) wrote…</i></summary><blockquote>

What happens to the rows that are _actually being dragged_, are those themable and distinctable from the non-dragged rows ?

There seems to be the `-drag-badge`, what should that be used for ?
</blockquote></details>

`-drag-badge` is the number that shows how many rows are being dragged. There is currently no theming for rows being dragged, should there be one?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/9219)*
<!-- Sent from Reviewable.io -->

wbadam committed:
Add v-draggable style name to doc

pleku commented:
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 76 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC1qtTbarAUOWtoEW0:-KjCMXvg0f8KRO1fXbRH:b7dlwyj) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L76)):*
<details><summary><i>Previously, wbadam (Adam Wagner) wrote…</i></summary><blockquote>

No it is applied when the extension is applied, removed when the extension is removed.

There is no style for 'component is being dragged', may make sense to add one.
</blockquote></details>

Well, user can currently do that themselves like is shown on our documentation example, so let's just leave it at that for now

---

*[documentation/advanced/advanced-dragndrop.asciidoc, line 225 at r1](https://reviewable.io:443/reviews/vaadin/framework/9219#-KjC65RY3DrbQ9DW_wNO:-KjCN1rCWz975HV3XRDE:b-ccwimm) ([raw file](https://github.com/vaadin/framework/blob/5f037f8509e12ede056b3f2781c7b9888a80e701/documentation/advanced/advanced-dragndrop.asciidoc#L225)):*
<details><summary><i>Previously, wbadam (Adam Wagner) wrote…</i></summary><blockquote>

`-drag-badge` is the number that shows how many rows are being dragged. There is currently no theming for rows being dragged, should there be one?
</blockquote></details>

I'll create a ticket

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/9219)*
<!-- Sent from Reviewable.io -->
